client/components/progress-bar/index.jsx

```diff
@@ -54,6 +58,16 @@ export default class ProgressBar extends PureComponent {
 		);
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		if ( this.state.allTimeMax < nextProps.value ) {
+			this.setState( { allTimeMax: nextProps.value } );
+		}
```

Comment:
Note that we don't need the `if` because we have a built-in function that does this…

```js
componentWillReceiveProps( { value } ) {
	this.setState( { progress: Math.max( this.state.progress, value ) } );
}
```

Reply:
ha! Good catch using this little one, I've updated now, thanks.
I haven't destructured though, since it's more self documented and if someone laters adds something here using `nextPress` again, it would be a smaller diff.

Reply:
that's fine, though it's no big deal to destructure early and let someone else change it later :)